doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -4830,6 +4830,25 @@ Place this ``user-add.json`` file in your current directory and run the followin
 
 Optionally, you may use a third query parameter "sendEmailNotification=false" to explicitly disable sending an email notification to the new user.
 
+Validate Builtin User Credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Validates the provided credentials to determine if the user can log in with them.
```

Comment:
Should address security here - which possibly depends on changes - see comment the API call.